src/login/logind-user.c

```diff
@@ -277,14 +279,14 @@ int user_save(User *u) {
 }
 
 int user_load(User *u) {
-        _cleanup_free_ char *realtime = NULL, *monotonic = NULL;
+        _cleanup_free_ char *realtime = NULL, *monotonic = NULL, *stopping = NULL;
         int r;
 
         assert(u);
 
         r = parse_env_file(NULL, u->state_file, NEWLINE,
                            "SERVICE_JOB", &u->service_job,
-                           "SLICE_JOB",   &u->slice_job,
+                           "STOPPING",    &stopping,
```

Comment:
we never start the slice explicitly anymore, as it now has StopWhenUnneeded= set. The unit is only implicitly pulled in now, by user@.service or by the session scopes, and automatically cleaned up by PID 1 when all depending units are gone. If we'd start the slice job manually too, then it would be terminated again right away unless a session scope or user@.service instance is also added.

Or in other words: it's a bit simpler now, we can drop code from logind to manage the slice manually, it's now managed entirely automatically by PID 1 itself, which is a good thing.

Reply:
the "slice_job" field in the User structure is entirely gone btw, this specific change is just an artifact of that.